salt/reviewer-suggestions/postgres.sls

```diff
@@ -0,0 +1,35 @@
+postgres-db-user:
+    postgres_user.present:
+        - name: {{ pillar.reviewer_suggestions.db.username }}
+        - encrypted: True
+        - password: {{ pillar.reviewer_suggestions.db.password }}
+        - refresh_password: True
+
+        {% if salt['elife.cfg']('cfn.outputs.RDSHost') %}
+        # remote psql
+        - db_user: {{ salt['elife.cfg']('project.rds_username') }}        
+        - db_password: {{ salt['elife.cfg']('project.rds_password') }}
+        - db_host: {{ salt['elife.cfg']('cfn.outputs.RDSHost') }}
+        - db_port: {{ salt['elife.cfg']('cfn.outputs.RDSPort') }}
+        {% else %}
+        - db_user: {{ pillar.elife.db_root.username }}
+        - db_password: {{ pillar.elife.db_root.password }}
+        {% endif %}
+        - createdb: True
+
+postgres-db-exists:
```

Comment:
no state requires `postgres-db-exists`, but it would be safer to make explicit the dependency on the database by requiring it in the first state in `init.sls` that writes to the database